analysers/experiments/cpu.py

```diff
@@ -35,11 +35,11 @@
 CassandraRDD = sc.cassandraTable(cassandraKeyspace, "trial_cpu")
```

Comment:
The changes are already in: https://github.com/benchflow/analysers/pull/51/files/308f5db903e9ea4515c647018d29c87835aa5db6..f8a0f6ddf9ed80eec8ce9b5d8e95f1a68fd2b363